README.md

```diff
@@ -19,38 +28,18 @@ A PowerDNS web interface with advanced features.
 - Limited API for manipulating zones and records
 - Full IDN/Punycode support
 
-## Running PowerDNS-Admin
-There are several ways to run PowerDNS-Admin. The easiest way is to use Docker.
-If you are looking to install and run PowerDNS-Admin directly onto your system check out the [Wiki](https://github.com/ngoduykhanh/PowerDNS-Admin/wiki#installation-guides) for ways to do that.
-
-### Docker
-This are two options to run PowerDNS-Admin using Docker.
-To get started as quickly as possible try option 1. If you want to make modifications to the configuration option 2 may be cleaner.
-
-#### Option 1: From Docker Hub
-The easiest is to just run the latest Docker image from Docker Hub:
-```
-$ docker run -d \
-    -e SECRET_KEY='a-very-secret-key' \
-    -v pda-data:/data \
-    -p 9191:80 \
-    ngoduykhanh/powerdns-admin:latest
-```
-This creates a volume called `pda-data` to persist the SQLite database with the configuration.
-
-#### Option 2: Using docker-compose
-1. Update the configuration   
-   Edit the `docker-compose.yml` file to update the database connection string in `SQLALCHEMY_DATABASE_URI`.
-   Other environment variables are mentioned in the [legal_envvars](https://github.com/ngoduykhanh/PowerDNS-Admin/blob/master/configs/docker_config.py#L5-L46).
-   To use the Docker secrets feature it is possible to append `_FILE` to the environment variables and point to a file with the values stored in it.   
-   Make sure to set the environment variable `SECRET_KEY` to a long random string (https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY)
-
-2. Start docker container
-   ```
-   $ docker-compose up
-   ```
-
-You can then access PowerDNS-Admin by pointing your browser to http://localhost:9191.
+## Deploying PowerDNS-Admin
+There are multiple ways to run the PowerDNS-Admin app. The recommended method is to use the official [Docker images](https://github.com/ngoduykhanh/PowerDNS-Admin/blob/master/docs/docker.md).
+
+If you would like to run PowerDNS-Admin directly on your machine or VM, check out the [Wiki](https://github.com/ngoduykhanh/PowerDNS-Admin/wiki#installation-guides) for additional information.
+
+Once you have deployed the app through one of the supported methods, You should be able to access the PowerDNS-Admin app by pointing your browser to http://localhost:8080.
+
+## Configuring PowerDNS-Admin
+
+The app has a [plethora of settings](https://github.com/ngoduykhanh/PowerDNS-Admin/blob/master/docs/settings.md) that may be configured through a number of methods. Check out the settings documentation [here](https://github.com/ngoduykhanh/PowerDNS-Admin/blob/master/docs/settings.md).
```

Comment:
Redundant link

Reply:
@jbe-dw I was always torn on whether or not this style of documenting is a "problem" per se. I'm always thinking about missed opportunity vs what seem like arbitrary standards at times. Thoughts?

Reply:
@jbe-dw The project README.md has been updated to clean this up.

Reply:
Great. The README is the entrypoint to the github repo and should be a good starting point for the user. It should remain concise and easy to understand. Someone interested in using the role will probably land here and see it as github displays it as an index. It should list functionalities, give examples and basic instruction for a quickstart.

If there are complex configurations or more detailed instructions need to be given, it should point to a complete "real" documentation. Either in a markdown file (like in /docs - appreciated for it person) or in read-the-docs (needed by user with small knowledge and not working in IT), or both.

Anyhow it is a showcase for the project. It should be clear, with no grammar errors, because if the starting page is messy, the project is probably too. It can be discussed later though

Reply:
I have taken your advice into consideration and retracted the change entirely for now. I will take the approach recommended once I can provide an updated outline file.